alpha/scripts/utils/updateEntriesRecordings.php

```diff
@@ -0,0 +1,109 @@
+<?php
+require_once('/opt/kaltura/app/tests/lib/KalturaClient.php');
+if ($argc < 5)
+{
+	die("Usage: php $argv[0] partnerId adminSecret serviceUrl simuliveVodFile \n");
+}
+$partnerId = $argv[1];
+$adminSecret = $argv[2];
+$url = $argv[3];
+$simuliveVodFile = $argv[4] ;
+
+$config = new KalturaConfiguration();
+$config->serviceUrl = $url;
+$client = new KalturaClient($config);
+$ks = $client->session->start(
+	$adminSecret,
+	'',
+	KalturaSessionType::ADMIN,
+	$partnerId,
+	86400,
+	'disableentitlement');
+$client->setKS($ks);
+
+$simuliveVodEntryIds = file ($simuliveVodFile) or die ('Could not read file'."\n");
+
+foreach($simuliveVodEntryIds as $simuliveEntryIdVodEntryId)
+{
+	$simuliveEntryIdVodEntryId = trim($simuliveEntryIdVodEntryId);
+	list($simuliveEntryId, $vodEntryId) = explode(',', $simuliveEntryIdVodEntryId);
+	try
+	{
+		$simuliveEntry = $client->baseEntry->get($simuliveEntryId);
+	}
+	catch(Exception $e)
+	{
+		print_r($e->getMessage() . ' entryId: ' . $simuliveEntryId ."\n");
+		continue;
+	}
+	if ($simuliveEntry)
+	{
+		$kLive = new KalturaLiveStreamEntry();
+		$kLive->recordedEntryId = $vodEntryId;
+		try
+		{
+			$client->baseEntry->update($simuliveEntryId, $kLive);
+			print_r('EntryId: ' . $simuliveEntryId . ' was updated with recordedEntryId: '. $vodEntryId . "\n");
+		}
+		catch(Exception $e)
+		{
+			print_r($e->getMessage() . ' entryId: ' . $simuliveEntryId ."\n");
+			continue;
+		}
+
+		$kLive = new KalturaLiveStreamEntry();
+		$kLive->displayInSearch = KalturaEntryDisplayInSearchType::SYSTEM;
+		try
+		{
+			$client->baseEntry->update($vodEntryId, $kLive);
+			print_r('EntryId: ' . $vodEntryId . ' was updated with displayInSearch: ' . KalturaEntryDisplayInSearchType::SYSTEM . "\n");
+		}
+		catch(Exception $e)
+		{
+			print_r($e->getMessage() . ' entryId: ' . $vodEntryId ."\n");
+			continue;
+		}
+		addCategoryEntryIds($client, $simuliveEntryId, $vodEntryId);
+	}
+}
+print_r("Done! \n");
+
+
+/*
+ * adding the category entries of simuliveEntryId to vodEntryId
+ */
+function addCategoryEntryIds($client, $simuliveEntryId, $vodEntryId)
+{
+	$categoryEntryFilter = new KalturaCategoryEntryFilter();
+	$categoryEntryFilter->entryIdEqual = $simuliveEntryId;
+	$categoryEntryFilter->statusEqual = KalturaCategoryEntryStatus::ACTIVE;

```

Comment:
per recent changes, we only alter the categories under the KMS tree: https://github.com/kaltura/mediaspace/blob/release/modulesCustom/core/eventplatform/reinventedit/models/Reinventedit.php#L207

@assafb this is according to the conversation we had with Yair where he (or you?) said they have a category out of KMS that holds all VODs.